integration-tests/publish.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"github.com/CESARBR/knot-cloud-storage/internal/config"
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/data"
+	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
+	"github.com/streadway/amqp"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"time"
+)
+
+type thing struct {
+	ID string
+	Data []entities.Payload
+}
+
+func failOnError(err error, msg string) {
+	if err != nil {
+		log.Fatalf("%s: %s", msg, err)
+	}
+}
+
+func setLogger(value string) *config.Logger {
+	return &config.Logger{Level: value}
+}
+
+func setPayload(id string, data []entities.Payload) *thing {
+	return &thing{ID: id,	Data: data,}
+}
+
+func setQuery(thingID string, sensorID string, order int, skip int64, take int64, startDate time.Time, finishDate time.Time) *entities.Query {
+	return &entities.Query{
+		ThingID: thingID,
+		SensorID: sensorID,
+		Order: order,
+		Skip: skip,
+		Take: take,
+		StartDate: startDate,
+		FinishDate: finishDate,
+	}
+}
+
+func showQueryResult(output []entities.Data) {
+	fmt.Println("THING_ID SENSOR_ID VALUE TIMESTAMP")
+	for i := range output {
+		fmt.Println(output[i])
+	}
+}
+
+func main() {
```

Comment:
As I said before, we must have only one `main`. So, I think you need to combine both files, decouple them in a way that could be easily reused (per function, per file, etc) and call them in the `script` file.

Reply:
I split the functions in different files, according to their meaning. I push these files in the same commit or I put in different ones?